front/lib/resources/vault_resource.ts

```diff
@@ -99,14 +99,21 @@ export class VaultResource extends BaseResource<VaultModel> {
   }
 
   static async listWorkspaceVaults(
-    auth: Authenticator
+    auth: Authenticator,
+    onlySystemAndGlobal = false
```

Comment:
This parameter does not seem used. Can we remove?

Reply:
It is used but I'll update the other functions as we talked about.